Consistence/Sniffs/Exceptions/ExceptionDeclarationSniff.php

```diff
@@ -46,6 +49,43 @@ public function process(PhpCsFile $phpcsFile, $classPointer)
 				$className
 			), $classPointer, self::CODE_NOT_ENDING_WITH_EXCEPTION);
 		}
+
+		$this->checkThatExceptionIsChainable($phpcsFile, $classPointer);
+	}
+
+	/**
+	 * @param \PHP_CodeSniffer\Files\File $phpcsFile
+	 * @param int $classPointer
+	 */
+	private function checkThatExceptionIsChainable(PhpCsFile $phpcsFile, int $classPointer)
+	{
+		$functionPointerToScan = $classPointer;
+		while (($functionPointer = TokenHelper::findNext($phpcsFile, T_FUNCTION, $functionPointerToScan)) !== null) {
+			$functionName = FunctionHelper::getName($phpcsFile, $functionPointer);
+			if ($functionName === '__construct') {
```

Comment:
Could you please refactor the `__construct` finding into a private method? This will help out the flow readability and reduce indentation.

Reply:
fixed